### ISSUE TITLE
Z-band streaming

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5094]
**Preferred term label:**

**Synonyms**
Z-band streaming

**Definition (free text, please give PubMed ID)**
Z-discs (Z-disk, Z-line, Z-band) delineate the lateral borders of sarcomeres and are the smallest functional units in striated muscle. The core of a Z-disc consists of actin filaments coming from adjacent sarcomeres which are crosslinked by alpha-actinin molecules. Z-discs, which are difficult to detect in conventional light microscopy, appear in the longitudinal view of electron microscopy as electron dense bands with varying sizes, ranging between 30 and 50 nm in fast muscle and between 100 and 140 nm in slow muscle and cardiac myocytes. In  a  large  number  of  neuropathic  and myopathic  conditions,  reactions  of  the  Z band can be seen,  most often in the guise of streaming or smearing (figure 4). The Z band is  then  no longer confined  to a narrow zone which  bisects  the  I  band.  It  may  extend across the I band or the entire sarcomere in a zigzag manner.  Focal thickening,  smudging, and  blurring  of  the  Z  band  takes  place concurrently.  Myofibrillar disorganization  is a frequent but not invariable accompanying change.
PMID:22028589   PMID:1180479 

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0004303

Probably add hierarchy for Z band

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4553032/
**Your nano-attribution (ORCID)**
